torchvision/models/shufflenetv2.py

```diff
@@ -19,7 +19,7 @@
 
 
 def channel_shuffle(x: Tensor, groups: int) -> Tensor:
-    batchsize, num_channels, height, width = x.data.size()
+    batchsize, num_channels, height, width = x.shape[0], x.shape[1], x.shape[2], x.shape[3]
```

Comment:
Thanks for spotting, this is the root cause of sizes exported as constant, correct?
Can you give a quick try if `x.size()` works? If so we can keep the change minimal. We should definitely add a section in onnx doc page about these known issues of things getting exported as constant :)

Reply:
Yes, x.size() also works and I've updated the code accordingly.